### DISCUSSION TITLE
Are PRs to add orgs still be accepted?

### DISCUSSION THREAD

**Discussion by shonfeder** [hng/tech-coops#79] [General]
I've noticed that there is growing backlog of PRs adding entries for co-ops. Is there a freeze on adding new co-ops? Do you need additional help maintaining this repo?

As PRs build up we increase the likelihood of merge conflicts, since they are all editing the same file. So I'm mainly concerned with ensuring we avoid creating more work for you or the contributors.

**Comment by trivigy** [comment]
@hng I will help you maintain and PR reviews. If you are willing to add contributors.

**Comment by hng** [comment]
Yes PRs are still accepted :) I've just had been extra busy the last few weeks while at the same time a bunch of new PRs come in. I will now check PRs more regularly and also @decentral1se is now helping with the PRs.

**Comment by shonfeder** [comment]
Greg news! Thanks @hng :)
